vertical-pod-autoscaler/pkg/utils/test/test_pod.go

```diff
@@ -53,6 +55,12 @@ func (pb *podBuilderImpl) WithLabels(labels map[string]string) PodBuilder {
 	return &r
 }
 
+func (pb *podBuilderImpl) WithAnnotations(annotations map[string]string) PodBuilder {
```

Comment:
Maybe it will be more usable if you made this AddAnnotation(name, value)?

Reply:
TBH, I want it to have the same signature as labels.
I'm ok with changing it to `AddAnnotation` as long as we also change `WithLabels` to `AddLabel`.

Reply:
Good point, I didn't notice the WithLabels. Let's leave as is then.